connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignorTest.java

```diff
@@ -137,6 +145,16 @@ public void testTaskAssignmentWhenWorkerJoins() {
         assertAssignment(2, 8, 0, 0, "worker1");
 
         // Second assignment with a second worker joining and all connectors running on previous worker
+        //
+        // assignment after this phase:
+        // W1: assignedConnectors:[], assignedTasks:[],
+        //     revokedConnectors:[C1], revokedTasks:[T1-0, T1-1, T1-2, T1-3]
```

Comment:
change (4): improve test readability by adding the final assignment for each phase of rebalance. It'll let other devs/users better understanding how the tests go and how the algorithm works.